Gemfile.lock

```diff
@@ -77,6 +77,9 @@ GEM
     rubocop-minitest (0.35.0)
       rubocop (>= 1.61, < 2.0)
       rubocop-ast (>= 1.31.1, < 2.0)
+    rubocop-obsession (0.1.5)
```

Comment:
I recommend using the latest version which fixes all bugs that were reported after the gem release:

```suggestion
    rubocop-obsession (0.1.9)
```